Makefile

```diff
@@ -115,16 +115,24 @@ deps-test:
 	$(PIP) install -U pip
 	$(PIP) install -r requirements_test.txt
 
+.PHONY: build install install-dev uninstall
+
+build:
+	$(PIP) install build setuptools_scm
+	$(foreach MODULE,$(BUILD_ORDER),$(PYTHON) -m build ./$(MODULE) &&) echo done
+# or use -n ?
+
 # (Re)install the tool
-install:
-#	$(PIP_INSTALL) $(BUILD_ORDER:%=./%/dist/ocrd$*(OCRD_VERSION)*.whl)
+install: #build
+#	$(PIP_INSTALL) $(BUILD_ORDER:%=./%/dist/ocrd*`$(PYTHON) -m setuptools_scm 2>/dev/null`*.whl)
 	$(foreach MODULE,$(BUILD_ORDER),$(PIP_INSTALL) ./$(MODULE) &&) echo done
 	@# workaround for shapely#1598
```

Comment:
Is there anything I can do to help resolve https://github.com/shapely/shapely/issues/1598?

Reply:
AFAIK the issue persists. (You can try `make -C ocrd_all test-workflow` in a GPU environment to see the crash, but make sure you comment out the workaround when installing/building everything.)

Reply:
OK, nothing much we can do and as long as the workaround is as simple as that, let's focus on more pressing issues.